### DISCUSSION TITLE
Permission for deleting form entries

### DISCUSSION THREAD

**Discussion by Jippeheijnen** [statamic/cms#4900] [Q&A]
Hi, I was wondering if there is a certain permission to delete form entries from my site. I know there is a ```- forms``` option in the roles.yaml, but that doesn't seem to work. I also know Super is able to manage them but I want to keep the other permissions that come with that separated. Thanks in advance

Edit: I am using statamic 2

**Comment by edalzell** [comment]
If it doesn't work it's likely not supported. V2 is not being added to, you should consider moving to v3 if possible.
